extra/release_tool.py

```diff
@@ -1864,7 +1864,9 @@ def tag_or_followed_branch(repo_git):
                             "-t",
                             gui_tag,
                             "-f",
-                            os.path.join(tmpdir, "Dockerfile.disclaimer"),
+                            os.path.join(tmpdir, "Dockerfile"),
+                            "--target",
+                            "disclaim",
```

Comment:
I am getting the following error:
```
Setting up temporary Git workspace...
--------------------------------------------------------------------------------
docker build -t mendersoftware/gui:tmp -f /home/lluis/northern.work/tmp_checkout/gui/Dockerfile --target disclaim /home/lluis/northern.work/tmp_checkout/gui

Ok to execute the above commands? yes

Sending build context to Docker daemon  127.9MB
Step 1/6 : FROM --platform=amd64 node:18.7.0-alpine AS base
...
...
...
Step 5/6 : FROM mendersoftware/gui:base AS disclaim
manifest for mendersoftware/gui:base not found: manifest unknown: manifest unknown
```

Reply:
Oh, you seem to be using an old docker version or not using buildkit... I'll adjust the command to use buildkit to account for such setups...

Reply:
This should now enable buildkit 🤞

Reply:
It is not enough (on my machine),

```
$ DOCKER_BUILDKIT=1 docker build -t mendersoftware/gui:tmp -f Dockerfile --target disclaim .
[+] Building 1.0s (3/3) FINISHED                                                    
 => [internal] load build definition from Dockerfile                           0.0s
 => => transferring dockerfile: 762B                                           0.0s
 => [internal] load .dockerignore                                              0.0s
 => => transferring context: 69B                                               0.0s
 => ERROR [internal] load metadata for docker.io/mendersoftware/gui:base       0.9s
------
 > [internal] load metadata for docker.io/mendersoftware/gui:base:
------
failed to solve with frontend dockerfile.v0: failed to create LLB definition: docker.io/mendersoftware/gui:base: not found
```

(also note DOCKER_BUILDKIT=1 instead of DOCKER_BUILDKIT 1) (edited)

Reply:
It works now :smiley: